### PR TITLE
📚 add tip about running karma js tests on mac

### PR DESCRIPTION
**Pull request by chrisjsewell**
This is how I got the npm tests to run on my Mac, not sure if there is a similar tip for Linux/Windows

### CODE REVIEW

**Comment by jayaddison** [comment]
> This is how I got the npm tests to run on my Mac, not sure if there is a similar tip for Linux/Windows

Yep, checked and confirmed that the same `FIREFOX_BIN` environment variable can be configured on Linux for situations when `firefox` isn't detected on the user's commandpath automatically :+1: 

I'd expect the same environment variable to be respected on Windows too, although I don't have an install of it locally to test that on.

**Review by jayaddison — APPROVED** [review]
Looks good to me.  We could generalize the message to other OSes.. or alternatively perhaps we should wait to find out whether people encounter those problems in practice for other platforms.

**Comment by picnixz** [comment]
For Windows, the way to set environment variables for a same line is a bit tricky (afaict from https://superuser.com/questions/223104/setting-and-using-variable-within-same-command-line-in-windows-cmd-exe).

While using `SET FIREFOX_BIN=...` would work, it would set the environment variable for the whole session, which sometimes you don't want to.

**Review by picnixz — APPROVED** [review]

File: doc/internals/contributing.rst
Comment (picnixz) [inline doc/internals/contributing.rst]:
```suggestion
  For Unix-based systems, you can specify the path to the Firefox binary using::
```

File: doc/internals/contributing.rst
Comment (picnixz) [inline doc/internals/contributing.rst:202]: Now that I remember, doesn't karma support other browsers?

Comment (chrisjsewell) [reply doc/internals/contributing.rst:202]: In `karma.conf.js` it is currently set up only as `browsers: ["Firefox"]`

Comment (picnixz) [reply doc/internals/contributing.rst:202]: I see. Before merging this one, maybe we should have a look at the karma deprecation issue that Jay just opened.

Comment (chrisjsewell) [reply doc/internals/contributing.rst:202]: I going to merge this for now, because its not clear how long it will take to migrate from karma, and theres no point to leaving this hanging.
Obviously when we migrate, we can/should update this section with any necessary information about how to run/maintain the JS tests

Comment (jayaddison) [reply doc/internals/contributing.rst:202]: I was thinking about this too, but hadn't mentioned it - it would be nice to add Chrome into the mix for JS testing too, and I'll file an issue with a feature request for that soon.

**Pull request merged**
